F/flex/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "flex"
+version = v"2.6.4"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/westes/flex/releases/download/v2.6.4/flex-2.6.4.tar.gz", "e87aae032bf07c26f85ac0ed3250998c37621d95f8bd748b31f15b33c45ee995")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+apk add gettext-dev
+cd ${WORKSPACE}/srcdir/flex-*
+./autogen.sh
+./configure --prefix=${prefix} --disable-static --enable-shared --host="$target"
+make -j${nproc} && make install
```

Comment:
I personally prefer to have commands on two different lines:
```suggestion
make -j${nproc}
make install
```
If one fails, it isn't immediately clear which one was the culprit